_posts/2020-05-26-full-text-search.md

```diff
@@ -59,16 +59,16 @@ Behind the scenes, this search feature uses [LunrJS](https://lunrjs.com), a soft
 
 Lunr builds an *inverted index* of all our lessons, which is essentially the same as an index at the back of a book. So each time you enter a search term, Lunr looks for term, finds all the lessons that is in, and then returns the lessons based on relevance of the term (which is calculated using an information retrieval algorithm called Okapi BM25 [https://en.wikipedia.org/wiki/Okapi_BM25](https://en.wikipedia.org/wiki/Okapi_BM25)).
```

Comment:
I more expected you to do something like:

```md
information retrieval algorithm called [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25)
```

for most of these links, so we don't have long urls hanging all over the place?

Reply:
very reasonable request that I just implemented 👍